gateway-js/src/index.ts

```diff
@@ -101,14 +100,6 @@ export function getDefaultGcsFetcher() {
     headers: {
       'user-agent': `apollo-gateway/${require('../package.json').version}`,
     },
-    retry: {
```

Comment:
Perhaps retrying is a reasonable thing to do with the new pipeline as well?

Reply:
3aa5233